crowbar_framework/lib/crowbar/repository.rb

```diff
@@ -0,0 +1,159 @@
+#
+# Copyright 2015, SUSE LINUX GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+module Crowbar
+  class Repository
+    attr_reader :platform, :id, :registry
+
+    class << self
+      def load!
+        @config = YAML.load_file(Rails.root.join("config/repos.yml"))
```

Comment:
Can we rename `@config` to something more explicit, like `@repos` or `@all_repos`?